bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -718,6 +718,23 @@ Full pathname to the inventory initialization control file.
 (Only relevant if FATES is on).
 </entry>
 
+<!-- This may eventually migrate to the dynamic_subgrid group-->
+<entry id="use_fates_luh" type="logical" category="physics"
+       group="elm_inparm" valid_values="" >
+If TRUE, enable use of land use state and transition data from luh_timeseries file.
+(Only valid for fates land use change runs, where there is a luh_timeseries file.)
```

Comment:
I think I'd remove this middle sentence here. I see the dependence the other way -- if this is set to true, then you have to have a fluh_timeseries file. So there isn't a need for this one.